vertx-db2-client/src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,223 @@
+= Reactive DB2 Client
+:PREPARED_PARAMS: `?`​
+
+The Reactive DB2 Client is a client for DB2 with a straightforward API focusing on
+scalability and low overhead.
+
+The client is reactive and non blocking, allowing to handle many database connections with a single thread.
+
+*Features*
+
+* Support for DB2 on Linux, Unix, and Windows
+* Limited support for DB2 on z/OS
+* Event driven
+* Lightweight
+* Built-in connection pooling
+* Prepared queries caching
+* Batch and cursor
+* Row streaming
+* RxJava 1 and RxJava 2
+* Direct memory to object without unnecessary copies
+* Java 8 Date and Time
+* HTTP/1.x CONNECT, SOCKS4a or SOCKS5 proxy support
+
+*Current limitations*
+
+* No TLS/SSL support
```

Comment:
Is this still true @aguibert ?

Reply:
it was correct at the time of merging the PR, and then later on I added SSL/TLS support and updated the doc accordingly:
https://github.com/eclipse-vertx/vertx-sql-client/commit/b6e4b6ce1d7bde529242234ddf876ce5a80ad13d

Reply:
Great, thanks!